pkg/controllers/provisioning/provisioner_test.go

```diff
@@ -1358,4 +1375,73 @@ var _ = Describe("Provisioner/Provisioning", func() {
 			})
 		})
 	})
+
+	Context("Resource Calculations", func() {
+		It("should schedule based on the max resource requests of containers and initContainers with sidecarContainers for order type1", func() {
```

Comment:
What is order type1?

Reply:
type1 is where we have sidecar containers at end after init sidecards and type2 is vice versa of it . its just naming the order since different order has different calculations